test/message/test_runner_output.out

```diff
@@ -475,11 +475,13 @@ ok 52 - callback async throw after done
       ---
       duration_ms: *
       ...
+    # 'only' does not work when node is started without the --test-only command-line option
```

Comment:
I don't know how much detail we want in the diagnostic, but, for example [this test](https://github.com/nodejs/node/blob/e9bf7c1259f2ee24c1b238d665f9c6c681af917d/test/message/test_runner_output.js#L295) uses `runOnly()`, not the `only` option. We probably don't need to differentiate though.